### ISSUE TITLE
[scatter] Refresh UI

### ISSUE THREAD

**Issue by beets** [datacommonsorg/website#399]
Needs a new design especially for selecting all the options. Potentially search based? The current selection UI is very confusing.

**Comment by tjann** [comment]
Some brainstorming/discussion here:

1a. Search box for parent place. When complete, have another search box for child place types.
1b. Search box for parent place. When complete, have a drop down select for child place types.

2. Move per capita and log options to the chart (like we did for per capita in Timeline)
3. When selecting variables, have 1 and 2 instead of check boxes where 1-> x and 2-> y. This can help get rid of the swap option without introducing a fourth toggle option on the chart.
4. I think the limit for number of people might have been Guha's request. It's not very generalized, but it is useful. We could surface it after selecting child place types. I'd reword to something like "Limit to places with population between ___ and ___." The edge case here is for places where we have other stats but not Total Population.

**Comment by tjann** [comment]
I would see if we can move all of these to the sidebar, right on top of the current PV tree options.

**Comment by tjann** [comment]
6. We currently use both "x" and "y" AND the StatVar names. Maybe it would be good to just use the StatVar names: https://screenshot.googleplex.com/7fy8LeNQfmJRssZ ? Not sure.

**Comment by tjann** [comment]
7. Instead of this error popup: https://screenshot.googleplex.com/4EVEFoyvbshAHy6 we can just list the ones user checked, and ask user to only select 2. This helps with poor memory ("which did I just check?") Note that if we take 3, this will need to modify a bit.
